src/Microsoft.Framework.CommandLineUtils/CommandLine/AnsiConsole.cs

```diff
@@ -9,22 +9,52 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer)
+        private AnsiConsole(TextWriter writer, IServiceProvider services)
         {
             Writer = writer;
-            OriginalForegroundColor = Console.ForegroundColor;
+
+            var runtimeEnv = (IRuntimeEnvironment)services.GetService(typeof(IRuntimeEnvironment));
+            var shouldParse = runtimeEnv.OperatingSystem == "Windows" || runtimeEnv.RuntimeType == "Mono";
+
+            if (shouldParse)
```

Comment:
I don't think `parse` is the right term here. Maybe `useColors` or something on those lines would be more appropriate